src/dotnet/commands/dotnet-sln/xlf/LocalizableStrings.cs.xlf

```diff
@@ -22,6 +22,11 @@
         <target state="translated">Přidá do souboru řešení jeden nebo více projektů.</target>
         <note />
       </trans-unit>
+      <trans-unit id="NoSolutionFolders">
+        <source>Add project to the root of the solution without creating any solution folders.</source>
```

Comment:
I have the overall consideration on the feature/switch name. That might cause me to later change my mind on this string :)

It's easy to forget about solution folders if you don't use them (adding is not the important part, where is). 

`Place project in root of the solution, rather than creating a solution folder.`

Reply:
@KathleenDollard I agree with your inputs on the naming. Something like `--place-project-in-root` or `--add-project-in-root` (or just simply `--in-root`, because you're specifying `dotnet sln add` anyways?) is much easier to understand, if it implies what it's doing and not, what it's not doing. I've changed it to `--place-project-in-root` for now (1a58044).

As for the resource string, I've updated that too in 1a58044.